mcp_server/src/mcp_server_neo4j_gds/server.py

```diff
@@ -102,6 +103,21 @@ async def handle_list_tools() -> list[types.Tool]:
                             "type": "object",
                         },
                     ),
+                    types.Tool(
+                        name="get_relationship_types",
+                        description="""Get relationship types in the database.""",
+                        inputSchema={
+                            "type": "object",
+                            "properties": {
+                                "nodeLabels": {
```

Comment:
This param means it can only get rel types between nodes of the same label?
As a result, it can never discover rel types connecting nodes of different labels and projecting them?